packages/bvaughn-architecture-demo/components/console/filters/FilterToggles.tsx

```diff
@@ -5,6 +5,7 @@ import { getMessages } from "@bvaughn/src/suspense/MessagesCache";
 import camelCase from "lodash/camelCase";
 import React, { Suspense, useContext, useMemo } from "react";
 import { ReplayClientContext } from "shared/client/ReplayClientContext";
+import Checkbox from "../../../../../src/ui/components/shared/Forms/Checkbox";
```

Comment:
This won't work. Surprised it didn't fail to compile.

Reply:
So I added a new Checkbox component under `packages/components`, but it's failing because NextJS can't handle importing CSS outside of the directory: 

<img width="420" alt="image" src="https://user-images.githubusercontent.com/2762082/183528731-d0a8dd25-c554-4f58-8215-04914e899f10.png">

Wondering if I should move it under your package for now and then we can share later?